ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -43,26 +43,41 @@
 #error "This file requires ARC support."
 #endif
 
-#define BLOG(verbose_level, format, ...)                  \
+namespace ledger {
+
+template <typename T>
+struct task_deleter {
+ private:
+  scoped_refptr<base::TaskRunner> task_runner;
+
+ public:
+  task_deleter()
+      : task_runner(base::SequencedTaskRunner::GetCurrentDefault()) {}
+  ~task_deleter() = default;
+
+  void operator()(T* ptr) const {
+    if (base::SequencedTaskRunner::GetCurrentDefault() != task_runner) {
+      task_runner->PostTask(FROM_HERE,
+                            base::BindOnce([](T* ptr) { delete ptr; }, ptr));
+    } else {
+      delete ptr;
+    }
+  }
+};
+
+template <typename T, typename... Args>
+std::unique_ptr<T, task_deleter<T>> make_task_ptr(Args&&... args) {
+  return {new T(std::forward<Args>(args)...), task_deleter<T>()};
+}
+
+}  // namespace ledger
+
+#define LLOG(verbose_level, format, ...)                  \
```

Comment:
Why the change to `LLOG`?

Reply:
Because `BLOG` was already defined and this was a redefinition that used our own client stuff… I agree though it'd be better to the replace the LedgerClient log method with Chromium logging directly since they all use Chromium logging anyways